app/src/main/java/com/github/sdp_begreen/begreen/models/event/Event.kt

```diff
@@ -1,13 +1,13 @@
 package com.github.sdp_begreen.begreen.models.event
 
 import com.github.sdp_begreen.begreen.firebase.RootPath
-import com.github.sdp_begreen.begreen.models.CustomLatLng
 import com.github.sdp_begreen.begreen.firebase.utils.CopyableWithId
+import com.github.sdp_begreen.begreen.models.CustomLatLng
 
 /**
  * Interface that define the base attribute that an Event should have
  */
-interface Event<T>: CopyableWithId<T> {
+interface Event<T> : CopyableWithId<T> {
```

Comment:
I didn't know that interfaces could declare attributes. Otherwise I would have declared `PhotoMetadata` as an interface instead of an abstract class. Thank you, I will think about it later